### ISSUE TITLE
[DataCap Application] Provide comprehensive solution for visualization of soccer fields - GoalV

### ISSUE THREAD

**Issue by twosson** [filecoin-project/filecoin-plus-large-datasets#1562]
### Data Owner Name

GoalV

### Data Owner Country/Region

China

### Data Owner Industry

Information, Media & Telecommunications

### Website

https://www.goalv.cn

### Social Media

```text
WeChat public account: 够威体育
```

### Total amount of DataCap being requested

5PiB

### Weekly allocation of DataCap requested

500TiB

### On-chain address for first allocation

f12cm6dpiw5jfo6zkyrhgd7vavpcgyuf77ut6zmfi

### Custom multisig

- [ ] Use Custom Multisig

### Share a brief history of your project and organization

```text
Let every game can be rewatched, so that every wonderful goal is recorded completely and from multiple angles. Let everyone share their wonderful moments with others conveniently and quickly!
```

### Is this project associated with other projects/ecosystem stakeholders?

No

### Describe the data being stored onto Filecoin

```text
Storing Sports Videos
```

### Where was the data currently stored in this dataset sourced from

Other

### If you answered "Other" in the previous question, enter the details here

```text
Aliyun OSS(Object Storage Service)
```

### How do you plan to prepare the dataset

IPFS, lotus, singularity, graphsplit

### Please share a sample of the data

```text
https://www.goalv.cn/video/GoalV-Panoramic-View_1.mp4
https://www.goalv.cn/video/Broadcast-Feed.mp4
https://www.goalv.cn/video/Players-data-collection.mp4
```

### Confirm that this is a public dataset that can be retrieved by anyone on the Network

- [X] I confirm

### What is the expected retrieval frequency for this data

Weekly

### For how long do you plan to keep this dataset stored on Filecoin

2 to 3 years

### In which geographies do you plan on making storage deals

Greater China, Asia other than Greater China

### How will you be distributing your data to storage providers

Cloud storage (i.e. S3)

### How do you plan to choose storage providers

Slack, Big data exchange, Partners

### How do you plan to make deals to your storage providers

Boost client, Lotus client

### Can you confirm that you will follow the Fil+ guideline

Yes

**Comment by cryptowhizzard** [comment]
Dear applicant,

Thank you for applying for datacap. As notary i am screening your application and doing due diligence.

Looking at your application i have some questions:

As you are brand new on Github and have no history of past applications. Applying for 5PB of datacap is a lot. One needs comprehensive knowledge of Filecoin, packing data and it's requirements and distribution of the data. Are you brand new in the Filecoin space or have you applied for datacap in the past on different Github account names?

Can you provide me with some KYC details per e-mail at kyc@dcent.nl. I would like to receive an e-mail from the original domainname / contact as listed on the website that refers to this application. 
Your application is for 5PB of datacap. Can you show us some visible proof from the data size and the storage you have there?

Can you provide us with a distribution plan supposably that you would be granted datacap to store?

As you can see here : https://github.com/filecoin-project/filecoin-plus-large-datasets it states:

> In order for a client and their dataset to be eligible:
> the dataset should be public, open, and mission aligned with Filecoin and Filecoin Plus. This also means that the data should be accessible to anyone > in the network, without requiring any special permissions or access requirement
> stored data should be readily retrievable on the network and this should be regularly verified (though the use of manual or automated verification > that includes retrieving data from various miners over the course of the DataCap allocation timeframe)

To make sure that you understand the rules and guidelines I would like to receive a detailed allocation plan where the amount of data and reputational SP's where you are going to store this data are included.

**Comment by twosson** [comment]
@cryptowhizzard hi, What information does Kyc, a Chinese company, need.

**Comment by cryptowhizzard** [comment]
Who you are ( Your name and where you live )
Who the people are / company is who are going to pack this data for you and store it on the network.
The email needs to come from the official domain name ( https://www.goalv.cn/ ) .... like from misterx@goalv.cn with the above information stated.

**Comment by twosson** [comment]
@cryptowhizzard The mail has been sent successfully. Please check.
The content is relatively large, including our application screen recording

**Comment by cryptowhizzard** [comment]
Hello,

Thanks for onboarding to the Filecoin network and doing the KYC.
This is a message to let you know that I received your e-mail.

I have reviewed your application and the documents included. I also checked the website and as far as i can see everything looks ok.

I do have some question here
You mention that your client wants to store the data. Who is your client then? Are you the data preparer or are you the SP? *please read below

For the technical side of things:
You mentioned you have a lot of data to store on the Filecoin network, consisting of mp4 video's. Are these video's copyright protected and are you allowed to store them on Filecoin?
Can you show us that by sending the documentation to the same address?

Doing data preparation and sending out the deals is a lot of work and it requires a lot of technical skills / knowledge and equipment. Are you going to do the data preparation yourself
or is someone else going to do this for you? What is his or her reputation and company name?

For the Filecoin + program itself please note that by storing data on the Filecoin network you get 10x the power ( QAP ) of a normal sector. In return for this to be eligible for this incentive there are important rules to abide:

- The data must be stored with multiple service providers. ( Not with one organization running multiple SP's but really independent organizations )
- The data must be distributed over multiple regions. (Running a VPN is not allowed to fake presence in the USA for example)
- The data must be retrievable by everyone in the community to check if you are really storing what you say you are storing. Running boost is highly recommended.
- The data must be in line with the mission of FIL+. That means that the data must be valuable for humanity. It can be scientific or other things that people are highly interested in and is unique.
- As data preparer you are allowed to store one copy yourself (if the client requests so for the sake of redundancy of the data). If you store your copy in China it means that the other organizations/SP must be outside China.

Let me know if you have questions at the moment. If not then i would like to create a data onboarding plan together with you. It consists of:

The entity that is going to do the data preparation for you, where and when. If you need recommendations i can give them to you.
The SP's you are going to store data with and their reputation. If you need recommendations i can give them to you.
When we do checks on the data stored and how and by who.
The retrieval frequency (how often does the data needs to be retrieved and from where)

Looking forward to your response!

**Comment by twosson** [comment]
1. Our customers refer to users of our applications.
2. We are data preparer.
3. As for the copyright issue, the data itself belongs to our own manufacture. The video equipment will not work until all the videos are approved by the players.

For the technical side of things:
We have set up a development team to dock with filecoin, so we will prepare data by ourselves. Storage data migration is a rigorous work, and cannot dock with a third party.

`As data preparer you are allowed to store one copy yourself (if the client requests so for the sake of redundancy of the data). If you store your copy in China it means that the other organizations/SP must be outside China.` I have a question about this. We need to store it in multiple regions in China This will help our server better access data. Of course, this is not a fatal problem. Our developers are following strn.network to help us access data

**Comment by simonkim0515** [comment]
## Datacap Request Trigger

#### Total DataCap requested 
> 5PiB

#### Expected weekly DataCap usage rate 
> 500TiB 

#### Client address
> f12cm6dpiw5jfo6zkyrhgd7vavpcgyuf77ut6zmfi

**Comment by cryptowhizzard** [comment]
Hi @twosson 

Your answers 1 / 2 and 3 are clear to me.

>For the technical side of things:
We have set up a development team to dock with filecoin, so we will prepare data by ourselves. Storage data migration is a rigorous work, and cannot dock with a third party.

Can you tell me why? As i don't see any history from you as datapreparer on GIT or references / track records I must advise you that data preparing is not something to take lightly.

> As data preparer you are allowed to store one copy yourself (if the client requests so for the sake of redundancy of the data). If you store your copy in China it means that the other organizations/SP must be outside China. I have a question about this. We need to store it in multiple regions in China This will help our server better access data. Of course, this is not a fatal problem. Our developers are following strn.network to help us access data

I am sorry but this is not in line with the Fil+ program. If you want to store solely in China then i advise you to move this application to Fil-E ( Enterprise ) and ask for an exception.  You can tag (@)kevzag if you want that.

If Fil-E is not what you want then we are still missing your storage allocation plan. Let's work on that ;-)

What is the SP id or id's that belong to you or are owned by you to store the original copy of the data on in China?
Who are the other Sp's with reputation you are going to store with in your first allocation and where are they located?

Thanks!

**Comment by twosson** [comment]
Hi, @cryptowhizzard !
Because of the Chinese Spring Festival, colleagues have a holiday, SP id needs to wait for you.

**Comment by Carohere** [comment]
Hi @twosson, 500TiB weekly allocation is way above the community-suggested value, any particular reason you are applying so?

**Comment by twosson** [comment]
hi, We are still thinking about whether if the total data exceeds 5PiB, it means that the excess part cannot participate in Plus+

**Comment by cryptowhizzard** [comment]
> Hi, @cryptowhizzard ! Because of the Chinese Spring Festival, colleagues have a holiday, SP id needs to wait for you.

Hello,

I would like to get this moving for you.
Last week we ( Dcent ) made some guidelines for us in order to put the first signature on LDN's for the first tranche of datacap.

You can find them [here](https://form.jotform.com/230232885049356) ( No need to fill out, i already have KYC from you )

_It is about this:_
Note: As a data preparer, you are allowed to store one copy yourself for redundancy at the request of the client. However, if your copy is stored in the USA, the other organizations/SPs must be outside your region. Dcent encourages applications with a minimum spread of 2 continents and 3 different organizations.

If you can help me with this before the weekend that would be highly appreciated.

**Comment by twosson** [comment]
@cryptowhizzard I try to finish it today. We can provide you with SP id in China about tomorrow.

**Comment by twosson** [comment]
@cryptowhizzard hi

`What is the SP id or id's that belong to you or are owned by you to store the original copy of the data on in China?`
f02024863

`Who are the other Sp's with reputation you are going to store with in your first allocation and where are they located?`
Do I need to determine the SP ID once? I plan to make some copies in Singapore, South Korea, Germany and the United States

What else do I need to do?

**Comment by Carohere** [comment]
hi @twosson, how many sps are confirmed now? and could you share their nodes, geographical locations, etc.?

> hi, We are still thinking about whether if the total data exceeds 5PiB, it means that the excess part cannot participate in Plus+

Not really, you can submit another application if needed. But note that only one address should be used for the same project.

**Comment by twosson** [comment]
@Carohere 

I can now confirm an SP ID: f02024863.
This SP is located in Shenzhen, China.

Other SPs have not been decided yet, and are considering data transmission schemes. Need to provide all SP IDs now?

**Comment by Carohere** [comment]
Given this's your first application, f02024863 is a brand-new node and no other sps have confirmed their participation. To move forward, [I would suggest you reduce the weekly allocation to 100TiB or less](https://filecoinproject.slack.com/archives/C01DLAPKDGX/p1661450182420319) and list more information about newly joined SPs in time.

**Comment by twosson** [comment]
@Carohere This SP has created a new ID for our business.

**Comment by cryptowhizzard** [comment]
> @Carohere
> 
> I can now confirm an SP ID: f02024863. This SP is located in Shenzhen, China.
> 
> Other SPs have not been decided yet, and are considering data transmission schemes. Need to provide all SP IDs now?

Yes, i suggest you do so. The reason is simple. We check if the SP' s you come up with do not belong to one organisation and secondly we check if they are not involved in "non compliant behaviour". It is in your own best interest to provide them upfront so i can give you a "go ahead" . These days i don't think you can find notary's anyway who sign without this information.

**Comment by twosson** [comment]
hi @cryptowhizzard @Carohere ,

The following are our regional SPs:

Shenzhen, China: f02024863
Penang, Malaysia: f02027847
UK: f02027952
Thailand: f02028138
California, USA: f02028326

**Comment by herrehesse** [comment]
@twosson hello there, looking at the SP’s and their consecutive numbers, it seems highly likely that a VPN is used to display the given regions.

Can you send proof of physical location of each SP?

**Comment by twosson** [comment]
@herrehesse ,
<img width="527" alt="image" src="https://user-images.githubusercontent.com/62056343/216754106-f3d68825-a093-4cb4-8f12-3dc5d2cf863e.png">
<img width="575" alt="image" src="https://user-images.githubusercontent.com/62056343/216754136-919ff7c3-7d70-4e1e-a47c-814afc6a3836.png">
<img width="667" alt="image" src="https://user-images.githubusercontent.com/62056343/216754146-56558efa-089c-4588-a98f-2e11acd1f2ea.png">
<img width="549" alt="image" src="https://user-images.githubusercontent.com/62056343/216754228-0bb33189-849c-4c5c-93ab-5bbdbabf88dc.png">

**Comment by herrehesse** [comment]
@twosson Thank you for your quick response! Will take a look at your data.

**Comment by twosson** [comment]
@herrehesse If necessary, I can provide you with ssh permission to verify

**Comment by twosson** [comment]
@herrehesse What else do I need to do? We hope to migrate our business data to Filecoin as soon as possible thank you.

**Comment by cryptowhizzard** [comment]
Hi @twosson 

Thank you for all your efforts. 

My (personal) policy is that i only sign on what i can verify. I can’t verify the reputation of newly created SP’s.

Normally i would give the benefit of the doubt, however, since all these miners are new, created at the same time and the numbers are subsequent i think it is highly likely that they are created by one organisation solely with the purpose to store this datacap. I don’t have their contact information to prove otherwise. This is against my (personal) policy where i sign off on new LDN’s where the data preparer can store maximum one copy himself. The rest has to go to 3 independent organisations on two different continents.

basically there are 2 options:

A: You find another notary who is convinced based on the information above to sign off. Slack -> application review is the place to be

B: You provide me with SP’s and contact information with existing reputation. You can do by email if preferred to kyc@dcent.nl . When checked ok I can sign of myself. You can also contact me on slack.

Thanks!

**Comment by twosson** [comment]
hi @cryptowhizzard ,
I give you ssh permission to view our network configuration Geographical distribution It can be fully explained that these are not from one organization Even if an organization can build Filecoin in different countries, I believe it is a good thing for FIecoin

We just want to start business as soon as possible. Our stadium data is too big to store.
We have made a lot of efforts to find SPs in different countries.

🙏🙏🙏

**Comment by BDE-io** [comment]
@twosson how did you make the determination to choose these SPs and do you have any existing relationship with any of them? 

https://github.com/filecoin-project/filecoin-plus-large-datasets/issues/1562#issuecomment-1416642355

**Comment by twosson** [comment]
hi @BDE-io,
Our company and Chimsen Co., Ltd. are strategic partners.
GoalV Co.,Ltd The main business is football match.
Chimsen Co.,Ltd The main business is IT intelligence.

The company has many branches in the world. 

Since our video is quite expensive on the public cloud, we decided to join the Filecoin ecosystem after being recommended by the company.

If you have any questions about data, I can provide public cloud data display.

**Comment by twosson** [comment]
<img width="1284" alt="image" src="https://user-images.githubusercontent.com/62056343/216812519-adcecb4c-60ae-485e-98bd-3730cc9856cd.png">
These data centers are built with the participation of the company. The technical system has developed for nearly 20 years It suits us very well

**Comment by twosson** [comment]
@herrehesse @cryptowhizzard @BDE-io @Carohere Hi,
There has been no news here for a long time. Do you need me to provide other information? Or would you like to help me get to the next milestone? I will prove my legitimacy in the first distribution.

**Comment by cryptowhizzard** [comment]
Hi @twosson 

It is my policy not to sign when i don't have KYC. Sorry!
Ps. Notary's are obliged to perform due diligence / KYC.

**Comment by cryptowhizzard** [comment]
In case you still want to consider -> https://form.jotform.com/230337462961356

**Comment by twosson** [comment]
@cryptowhizzard I made a few changes to the SP selection and now I have submitted the form.

**Comment by cryptowhizzard** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceclrxorbusmw6ke4hy4ikd7gxml727ozwu3sorrnxrbaxlwxpn3kk 
#### Address 
> f12cm6dpiw5jfo6zkyrhgd7vavpcgyuf77ut6zmfi
#### Datacap Allocated
> 250.00TiB
#### Signer Address
> f1krmypm4uoxxf3g7okrwtrahlmpcph3y7rbqqgfa
#### Id
> 062ed3a7-ec17-4a63-871d-72a77568fb1b
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceclrxorbusmw6ke4hy4ikd7gxml727ozwu3sorrnxrbaxlwxpn3kk

**Comment by cryptowhizzard** [comment]
Well received!

Looking forward to your first milestone.

**Comment by twosson** [comment]
![411677304731_ pic](https://user-images.githubusercontent.com/62056343/221341723-a4e157eb-3794-4375-9632-5425f90977fc.jpg)

**Comment by twosson** [comment]
I have already sent an email to the fil+ official, is that OK?

**Comment by NDLABS-OFFICE** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzaced62ksdbenzesh2lln3dz4wk5qn5oyxti3id7m2mo4xhw5qzapeew 
#### Address 
> f12cm6dpiw5jfo6zkyrhgd7vavpcgyuf77ut6zmfi
#### Datacap Allocated
> 250.00TiB
#### Signer Address
> f1yayfsv6whu3rheviucvventj3y6t542xfpb47ei
#### Id
> 062ed3a7-ec17-4a63-871d-72a77568fb1b
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaced62ksdbenzesh2lln3dz4wk5qn5oyxti3id7m2mo4xhw5qzapeew

**Comment by kernelogic** [comment]
Retrieval test OK

<img width="940" alt="image" src="https://user-images.githubusercontent.com/3711825/225218767-664861ba-d80b-4634-bfbf-9af28da897b3.png">

**Comment by kernelogic** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacecmbeem6sxer4fjgbc6jpim4sqgthu43l6g7fhlw5e5nqc2ofamvk 
#### Address 
> f12cm6dpiw5jfo6zkyrhgd7vavpcgyuf77ut6zmfi
#### Datacap Allocated
> 500.00TiB
#### Signer Address
> f1yjhnsoga2ccnepb7t3p3ov5fzom3syhsuinxexa
#### Id
> 718111af-43c5-4e60-8244-acd88e686b1d
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacecmbeem6sxer4fjgbc6jpim4sqgthu43l6g7fhlw5e5nqc2ofamvk

**Comment by igoovo** [comment]
The retrieval is successful.
![WechatIMG8](https://user-images.githubusercontent.com/103082537/225228693-8b5c87b4-1d9f-44fc-b36b-e6a5ced780c3.jpeg)

**Comment by igoovo** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceduikmuthhycwe3p3k6smfdgzajn2q4rewkyq6lrxlbs5fbzrr7d6 
#### Address 
> f12cm6dpiw5jfo6zkyrhgd7vavpcgyuf77ut6zmfi
#### Datacap Allocated
> 500.00TiB
#### Signer Address
> f1shnsfayxqll77svffaxnjenms7bbbysbqcatrpy
#### Id
> 718111af-43c5-4e60-8244-acd88e686b1d
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceduikmuthhycwe3p3k6smfdgzajn2q4rewkyq6lrxlbs5fbzrr7d6

**Comment by YuanHeHK** [comment]
./bin/lotus client retrieval-ask f02027952 bafybeidq7pmdrxc5ffblx4u2gz2jxmlyimwlxejnonk3xpgrn33yqfj4y4
Ask: f02027952
Unseal price: 0 FIL
Price per byte: 0 FIL
Payment interval: 1 MiB
Payment interval increase: 1 MiB
Size: 31.75 GiB
Total price for 34091302912 bytes: 0 FIL

./bin/lotus client retrieval-ask f02024863 bafybeictleenfoumnx2mrpqft6goc77deoc262ughou2hd7fy26spgctry
Ask: f02024863
Unseal price: 0 FIL
Price per byte: 0 FIL
Payment interval: 1 MiB
Payment interval increase: 1 MiB
Size: 31.75 GiB
Total price for 34091302912 bytes: 0 FIL

The retrieval query that randomly checked both SPs looked healthy. I will support this time.

**Comment by twosson** [comment]
checker:manualTrigger

**Comment by YuanHeHK** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacec4jg2k5aoetr3jwjg4ht77snzubidpsmtctbfzwhcjsit5swkfyi 
#### Address 
> f12cm6dpiw5jfo6zkyrhgd7vavpcgyuf77ut6zmfi
#### Datacap Allocated
> 1000.00TiB
#### Signer Address
> f1fg6jkxsr3twfnyhdlatmq36xca6sshptscds7xa
#### Id
> c5797314-9964-402f-9406-d50622794c4d
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacec4jg2k5aoetr3jwjg4ht77snzubidpsmtctbfzwhcjsit5swkfyi

**Comment by NDLABS-OFFICE** [comment]
checker:manualTrigger

**Comment by liyunzhi-666** [comment]
checker:manualTrigger

**Comment by liyunzhi-666** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacebwqnbdvl4cqtxemcqck2vxtmwfmxdps7ln6xgnini4rpe4nvs3bw 
#### Address 
> f12cm6dpiw5jfo6zkyrhgd7vavpcgyuf77ut6zmfi
#### Datacap Allocated
> 1000.00TiB
#### Signer Address
> f1pszcrsciyixyuxxukkvtazcokexbn54amf7gvoq
#### Id
> c5797314-9964-402f-9406-d50622794c4d
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacebwqnbdvl4cqtxemcqck2vxtmwfmxdps7ln6xgnini4rpe4nvs3bw

**Comment by kernelogic** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceaxvoszwhh6e6uuvdnbx2kbhinwmjtz6qazy2angqpmvzndilmsf6 
#### Address 
> f12cm6dpiw5jfo6zkyrhgd7vavpcgyuf77ut6zmfi
#### Datacap Allocated
> 1.95PiB
#### Signer Address
> f1yjhnsoga2ccnepb7t3p3ov5fzom3syhsuinxexa
#### Id
> 9daa19bf-7e45-4b97-9ef5-22827cc943f3
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceaxvoszwhh6e6uuvdnbx2kbhinwmjtz6qazy2angqpmvzndilmsf6

**Comment by kernelogic** [comment]
DD performed a few days ago, see above.

**Comment by a1991car** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzacedelawmspwx2gybo6m4ydat25esdtujpvtpw6r26wnkllq5o4ic22 
#### Address 
> f12cm6dpiw5jfo6zkyrhgd7vavpcgyuf77ut6zmfi
#### Datacap Allocated
> 1.95PiB
#### Signer Address
> f1qnumecdypgrbaebtkdfjnwt5ndacadcuas3deiq
#### Id
> 9daa19bf-7e45-4b97-9ef5-22827cc943f3
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacedelawmspwx2gybo6m4ydat25esdtujpvtpw6r26wnkllq5o4ic22

**Comment by a1991car** [comment]
![微信图片_20230320163510](https://user-images.githubusercontent.com/32447611/226286833-edfdd5eb-e83c-4ab8-a169-caa2b7c45aec.jpg)

**Comment by twosson** [comment]
checker:manualTrigger

**Comment by twosson** [comment]
checker:manualTrigger

**Comment by twosson** [comment]
checker:manualTrigger

**Comment by 1ane-1** [comment]
![818b1f5c5a7342ef36468bfc44a48712](https://user-images.githubusercontent.com/91301211/233534887-2575cf55-18f3-4f64-a3b7-80076cf40087.png)
Retrieval and CID report looks great, this ground i will support it.

**Comment by 1ane-1** [comment]
## Request Proposed
Your Datacap Allocation Request has been proposed by the Notary
#### Message sent to Filecoin Network
>bafy2bzacebhdgagtynkwfquke5yfwwhwjh5cvabelzzga7hq2zvtklqnntity 
#### Address 
> f12cm6dpiw5jfo6zkyrhgd7vavpcgyuf77ut6zmfi
#### Datacap Allocated
> 1.34PiB
#### Signer Address
> f1mdk7s2vntzm6hu35yuo6vjubtrpfnb2awhgvrri
#### Id
> 
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzacebhdgagtynkwfquke5yfwwhwjh5cvabelzzga7hq2zvtklqnntity

**Comment by xingjitansuo** [comment]
## Request Approved
Your Datacap Allocation Request has been approved by the Notary
#### Message sent to Filecoin Network
>bafy2bzaceaku5r3wvukvliu57vpjsyahzowgg7kzuz5fvwnyoju2q24sb3t6s 
#### Address 
> f12cm6dpiw5jfo6zkyrhgd7vavpcgyuf77ut6zmfi
#### Datacap Allocated
> 1.34PiB
#### Signer Address
> f1a2lia2cwwekeubwo4nppt4v4vebxs2frozarz3q
#### Id
> 
#### You can check the status of the message here: https://filfox.info/en/message/bafy2bzaceaku5r3wvukvliu57vpjsyahzowgg7kzuz5fvwnyoju2q24sb3t6s
